source/content/guides/secrets/02-basic-concepts.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Pantheon Secrets Guide
+subtitle: Basic Concepts
+description: Gaining familiarity with some concepts about Pantheon Secrets will help you make the most of this feature.
+contributors: [stovak]
+contenttype: [guide]
+innav: [true]
+categories: [secrets]
+cms: [drupal, wordpress]
+audience: [development]
+product: [secrets]
+integration: [--]
+tags: [reference, cli, local, terminus, workflow]
+permalink: docs/guides/secrets/basic-concepts
+reviewed: "2024-05-01"
+---
+<dl>
+
+<dt>Secret</dt>
+
+<dd>
+
+A key-value pair that should not be exposed to the general public, typically something like a password, API key, or other sensitive information that should not be added to version control.
+
+</dd>
+
+<dt>Secret type<sup>1</sup></dt>
+
+<dd>
+This represents how the secret is used.  A secret can only have one type.
+
+Current types are:
+
+  * `runtime`: This secret type can be retreived directly from your application code.  This is the recommended type if you want your application to be able to use the secret while it's operating.
```

Comment:
Maybe name-drop `pantheon_get_secret()` here, so that folks who are skimming the documentation get the information they need sooner (and do not decide to just use type env with "getenv()")

Reply:
Added